Website/Import/WpImportManager.cs

```diff
@@ -8,11 +8,48 @@
 using Sitecore.Modules.WeBlog.Managers;
 using System.Xml;
 using System;
+using Sitecore.Resources.Media;
 
 namespace Sitecore.Modules.WeBlog.Import
 {
     public static class WpImportManager

```

Comment:
I know this is existing code, but can we make this non-static? Make it easier to test.